src/pages/Experiments/TasksMenuBlock/TasksMenu/DraggableItem/index.js

```diff
@@ -40,6 +43,10 @@ const MenuItem = ({
       </div>
 
       <div className='draggable-item-content-text'>{children}</div>
+
+      <div className='draggable-item'>
```

Comment:
You can remove the `draggable-item` class, because this class was created for this component:

![image](https://user-images.githubusercontent.com/36794274/152403872-65d0ef55-17b3-4d0b-8e1a-3cb9cb9c22ee.png)